.github/PULL_REQUEST_TEMPLATE/bug_fix_or_feature.md

```diff
@@ -0,0 +1,59 @@
+---
+name: "Bug fix or new feature"
+description: ''
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Description
+Please include a summary of the changes introduced by this PR.
+Additionally include any relevant motivation and context. List any
+dependencies that are required for this change, other PRs and
+associated issues.
+
+## Associated Pull Requests:
+- Make a list (with links!)
+- [Like this PR#691](https://github.com/OP2/PyOP2/pull/691)
+
+## Fixes Issues:
+- Make a list of issues (with links!)
+- [Like this Issue#2824](https://github.com/firedrakeproject/firedrake/issues/2824)
+
+Ensure above issues are linked, so they are closed when this PR is merged.
```

Comment:
```suggestion
If issues are fixed by this PR, prepend them with the word "fixes", so they are automatically closed when this PR is merged.
```

Reply:
Reminder that `fixes #123, #456` only autocloses `#123` last time I checked, so it needs something like `fixes #123, fixes #456` ?

Reply:
Ah yes